### DISCUSSION TITLE
Addresses formats for load balancing

### DISCUSSION THREAD

**Discussion by uzador** [tikv/grpc-rs#601] [Q&A] [answered]
I didn't figure out myself how to configure addresses on my client so I'm asking here:

Code is here
`https://github.com/tikv/grpc-rs/blob/master/tests-and-examples/examples/load_balancing/client.rs#L26`

.connect("**ipv4**:127.0.0.1:50051,127.0.0.1:50052");

Above works fine. My question: are there any other formats (with ipv6: for an instance)?

**Comment by BusyJay** [comment, marked as answer]
grpc-rs is a wrapper around grpc c core. Technically it's supposed to support everything c core supports. You can check the docs of gRPC here: https://github.com/grpc/grpc/blob/master/doc/naming.md.

**Reply by uzador** [reply]
Cool, @BusyJay thanks for the quick response! :)
